### PR TITLE
Development

### PR DESCRIPTION
**Pull request by Moraxan**
Weekly pull to main from development

### CODE REVIEW

**Review by daniellangton91 — APPROVED** [review]
Shouldn't be any problems seeing that the dev branch is working.

**Review by Chab93 — APPROVED** [review]
Maybe wait with merge until the current other 2 pulls are merged with dev before merging with main?

**Comment by Moraxan** [comment]
Agreed. Will click approve when the next pulls drop in.

**Pull request merged**
